AdtSampleApp/SampleFunctionsApp/AdtUtilities.cs

```diff
@@ -63,8 +61,9 @@ public static async Task UpdateTwinProperty(DigitalTwinsClient client, string tw
             // Update twin property
             try
             {
-                List<object> twinData = new List<object>();
-                twinData.Add(new Dictionary<string, object>() {
+                var twinData = new List<object>();
+                twinData.Add(new Dictionary<string, object>
```

Comment:
Should this be changed to use the JsonPatch helper class in the SDK?

Reply:
I could do that, sure.

Reply:
Actually, this one is harder because the caller passes in the operation. I did update the subsequent one, though.